consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -1168,6 +1169,11 @@ func (x *XDPoS_v2) SetNewRoundFaker(newRound utils.Round, resetTimer bool) {
 	}
 	x.currentRound = newRound
 }
+func (x *XDPoS_v2) ProcessQCFaker(chain consensus.ChainReader, qc *utils.QuorumCert) error {
```

Comment:
I think just call `ProcessQC` but add comment this function is for test only.

Reply:
Why do we even need this function?

Reply:
For mining test, e.g., already have 899 and want to mine 900, we need a node to have QC of 899, round of 900, etc.